docs/publishing/README.md

```diff
@@ -20,3 +20,27 @@ And you're all set up.
 ## Example
 The [Set-Game Example](https://github.com/open-wc/example-vanilla-set-game/) has the default publishing via storybook on netlify.
 You can see the finished page here: [https://example-set-game-open-wc.netlify.com/](https://example-set-game-open-wc.netlify.com/).
+
+## Serving With Apache HTTP Server
+
+The `dist` directory created by `npm run build` can be deployed on a local web server. These directions are for the [Apache HTTP Server](http://httpd.apache.org/), but should be adaptable to other web servers.
+
+- Build command: `npm run build`
+- Copy `dist` to the desired location: `sudo cp -R dist /Library/WebServer/Documents/myapp`
+- Add a `<VirtualHost>` directive to `httpd.conf`, either directly or by include:
```

Comment:
include? or including?

Reply:
Hm... The Apache config line would be something like `Include /private/etc/apache2/extra/httpd-myapp.conf`, so I said include. Another way to say it might by "or by an `Include` directive"

Reply:
"or by an `Include` directive" sounds good :)